swagger/dfc.yaml

```diff
@@ -234,6 +234,31 @@ paths:
                     "@type": "@id"
                 dfc-b:stockLimitation: '3'
                 dfc-b:sku: new-sku
+  "/api/dfc/enterprise_groups/{id}":
+    get:
+      summary: Show groups
+      parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: string
+      tags:
+      - EnterpriseGroups
+      responses:
+        '200':
+          description: successful
+          content:
+            application/json:
+              examples:
+                test_example:
+                  value:
+                    "@context": https://www.datafoodconsortium.org
+                    "@id": http://test.host/api/dfc/enterprise_groups/60000
+                    "@type": dfc-b:Enterprise
+                    dfc-b:hasName: Enterprise group
+                    dfc-b:hasDescription: this is a group
+                    dfc-b:VATnumber: ''
```

Comment:
I noticed that you didn't pass `VATnumber` to `DataFoodConsortium::Connector::Enterprise`, but I guess it's there by default.
Still, I would have expected it to be nil, and perhaps be trimmed from the output.

Reply:
> I noticed that you didn't pass VATnumber to DataFoodConsortium::Connector::Enterprise, but I guess it's there by default.
Still, I would have expected it to be nil, and perhaps be trimmed from the output.

Me, too. Some properties are initialised with `nil` and then omitted. This one seems to be initialised with an empty string and that is a value. We could raise an issue with Maxime but I don't think it's worth it right now.